secret_service/secret_service.go

```diff
@@ -42,22 +49,31 @@ func NewSecret(session dbus.ObjectPath, secret string) Secret {
 // SecretService is an interface for the Secret Service dbus API.
 type SecretService struct {
 	*dbus.Conn
-	object dbus.BusObject
+	object      dbus.BusObject
+	KeyringName string
 }
 
 // NewSecretService inializes a new SecretService object.
-func NewSecretService() (*SecretService, error) {
+func NewSecretService(keyringName string) (*SecretService, error) {
```

Comment:
A breaking change in libraries is not a great thing to do.
Please let the signature like this and add another func:
```
func WithKeyringName(name string) (*SecretService, error) {
    s,err := NewSecretService()
    if err != nil { return nil, err }
    s.KeyringName = name
    return s, nil
}
```